docker/scripts/entrypoint.sh

```diff
@@ -148,10 +167,27 @@ if [ "${DEV_MODE:-false}" != "true" ]; then
   validate_and_download "ZKV_CONF_CHAIN" "ZKV_SPEC_FILE_URL"
 fi
 
-ZKV_SECRET_PHRASE_PATH=${ZKV_SECRET_PHRASE_PATH:-"/data/config/secret_phrase.dat"}
-ZKV_NODE_KEY_FILE=${ZKV_NODE_KEY_FILE:-"/data/config/node_key.dat"}
-echo "  ZKV_SECRET_PHRASE_PATH=${ZKV_SECRET_PHRASE_PATH}"
-echo -e "  ZKV_NODE_KEY_FILE=${ZKV_NODE_KEY_FILE}\n"
+# Creating node key file if node key is provided
+if [ -n "${ZKV_NODE_KEY}" ]; then
+  ZKV_NODE_KEY_FILE="/tmp/node_key.dat"
```

Comment:
Maybe we'd like to have the option of this path being user controlled. You could get the value from the env var if set and alternatively set a default path on L151.

/tmp is also not guaranteed to be in memory, by default with Docker it is disk backed. See:
```
docker run --rm ubuntu:jammy findmnt -n -o SOURCE --target /tmp/
/dev/sdf1[/subvol/var/lib/docker/btrfs/subvolumes/a77d7edc4c8e51cbadb906fe0c4b0e88208c9e13027889bee81b6348af56da87]
```

`/dev/shm` would be the safer alternative if we never want to write this file to disk.

Reply:
I was thinking of more enforcing this location to be under `tmpf` volume which will be manadatory added to the compose project

```    
tmpfs:
      - /tmp
 ```
Therefore, not giving USER's an option to save it somewhere else rather than RAM DISK.

@cronicc let me know what do you think